presto-native-execution/presto_cpp/main/functions/tests/FunctionMetadataTest.cpp

```diff
@@ -40,24 +40,48 @@ class FunctionMetadataTest : public ::testing::Test {
     functionMetadata_ = getFunctionsMetadata();
   }
 
+  static std::string getFunctionKindAndParams(const json& entry) {
+    auto str = entry["functionKind"].get<std::string>();
+    if (entry["paramTypes"].is_array()) {
+      for (auto const& paramType : entry["paramTypes"]) {
+        str += "-" + paramType.get<std::string>();
+      }
+    }
+    return str;
+  }
+
+  static void sortMetadataList(json::array_t& list) {
+    for (auto& metadata : list) {
+      for (auto const& [key, val] : metadata.items()) {
```

Comment:
Can you explain the relevant of this check in a comment.

Reply:
Done